pom.xml

```diff
@@ -32,13 +32,12 @@
     <packaging>jar</packaging>
     <version>1.0-SNAPSHOT</version>
 
-    <name>Brooklyn CloudFoundry parent project</name>
+    <name>Brooklyn CloudFoundry project</name>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <project.entry>org.apache.brooklyn.cloudfoundry.BrooklynCloudFoundryMain</project.entry>
-        <brooklyn.version>0.10.0-SNAPSHOT</brooklyn.version> <!-- BROOKLYN_VERSION -->
+        <brooklyn.version>0.10.0-SNAPSHOT</brooklyn.version>
+        <cloudfoundry.client.version>1.1.2</cloudfoundry.client.version>
```

Comment:
Sorry, I am accustomed to use this version. Robert also pointed to this some time ago. 
Updating to 1.1.3